lib/Dialect/FIRRTL/Export/FIREmitter.cpp

```diff
@@ -531,6 +534,21 @@ void Emitter::emitDeclaration(LayerOp op) {
   });
 }
 
+/// Emit an option declaration.
+void Emitter::emitDeclaration(OptionOp op) {
+  startStatement();
+  ps << "option " << PPExtString(op.getSymName()) << " :";
```

Comment:
I think this is missing a legalize:

```suggestion
  ps << "option " << PPExtString(legalize(op.getSymName())) << " :";
```